functions/src/controllers/nominationController.ts

```diff
@@ -0,0 +1,45 @@
+import {Router} from "express";
+import {Event, Nomination, Player} from "../bdd/entities";
+import {BadRequestError} from "../errors";
+import {BddService} from "../services/BddService";
+import {authVerification} from "./commonMiddlewares/authMiddlewares";
+
+const router = Router();
+
+router.post("/create", authVerification, async function (req,res,next) {
+
+    const player = <Player> await BddService.playerHandler.findPlayerById(req.body.playerId);
+    const target = <Player> await BddService.playerHandler.findPlayerById(req.body.targetId);
+    
+    if(target === null || player === null) {
+        throw new BadRequestError("Invalid Player Data");
+    }
+
+    const event = <Event> await BddService.eventHandler.findEventById(req.body.eventId);
+
+    if(event === null) {
+        throw new BadRequestError("Invalid Event Data");
+    };
+
+    let nomination = await BddService.nominationHandler.createNomination(player, target, event);
+
+    if (nomination != null) {
+        return res.status(200).send(nomination);
+    }
+    else {
+        throw new BadRequestError("Invalid Nomination Data");
+    } 
```

Comment:
Après tu peux simplifier aussi, ça m'est passé sous le nez tout à l'heure j'ai pas fais gaffe, mais note pour toi plus tard : 

On évite de faire des else, tu peux simplement faire des : 
```suggestion
    if (nomination == null) {
        throw new BadRequestError("Invalid Nomination Data");
    }
    
    return res.status(200).send(nomination);
```